### PR TITLE
sys-kernel/rt-sources: version bump to {3.18.59_p65,4.4.75_p88,4.9.35…

### PR DESCRIPTION
**Pull request by lilrc**
…_p25,4.11.8_p5}

Package-Manager: Portage-2.3.6, Repoman-2.3.1

### CODE REVIEW

**Comment by aliceinwire** [comment]
There is some problem on moving the ebuild to EAPI="6" ?

File: sys-kernel/rt-sources/rt-sources-3.18.59_p65.ebuild [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="5"
```
Comment (aliceinwire) [inline sys-kernel/rt-sources/rt-sources-3.18.59_p65.ebuild]: check if we can move to EAPI="6"

File: sys-kernel/rt-sources/rt-sources-4.11.8_p5.ebuild [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="5"
```
Comment (aliceinwire) [inline sys-kernel/rt-sources/rt-sources-4.11.8_p5.ebuild]: check if we can move to EAPI="6"

File: sys-kernel/rt-sources/rt-sources-4.4.75_p88.ebuild [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="5"
```
Comment (aliceinwire) [inline sys-kernel/rt-sources/rt-sources-4.4.75_p88.ebuild]: check if we can move to EAPI="6"

File: sys-kernel/rt-sources/rt-sources-4.9.35_p25.ebuild [hunk 4]
```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="5"
```
Comment (aliceinwire) [inline sys-kernel/rt-sources/rt-sources-4.9.35_p25.ebuild]: check if we can move to EAPI="6"

**Comment by lilrc** [comment]
@aliceinwire Bumped to EAPI 6 and fixed Gentoo bug 624110.

**Pull request closed without merging**